doc/src/equality.md

```diff
@@ -2,10 +2,18 @@
 
 When testing for equality in Slosh, there are several functions that can be used.
 Consider invocations like
-`(= 1 2)`,
-`(eq? 1 2)`,
-`(equal? 1 2)`, and
-`(assert-equal 1 2)`
+```slosh
```

Comment:
to use the pre-processor you just need to append the string slosh to the appropriate code blocks